tests/Picard_compressible/screen-output

```diff
@@ -6,32 +6,32 @@ Number of degrees of freedom: 53,952 (25,344+3,264+12,672+12,672)
    Solving temperature system... 0 iterations.
    Skipping plastic_strain composition solve because RHS is zero.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 65+0 iterations.
+   Solving Stokes system... 54+0 iterations.
       Relative nonlinear residual (Stokes system) after nonlinear iteration 1: 1
 
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 73+0 iterations.
-      Relative nonlinear residual (Stokes system) after nonlinear iteration 2: 0.00886673
+   Solving Stokes system... 58+0 iterations.
```

Comment:
I didnt see this at first, but I would guess it is caused by https://github.com/geodynamics/aspect/issues/5114. This test contains `rand_seed` in the initial condition and with the 9.4 tester this will be a constant temperature instead of a random fluctuation. Should we fix the test to use a different initial condition? Or do we just wait for deal.II 9.5?